NickvisionMoney.GNOME/Views/MainWindow.cs

```diff
@@ -278,32 +301,65 @@ private async void OnWindowShow(Gtk.Widget sender, EventArgs e)
     private void OnNewAccount(Gio.SimpleAction sender, EventArgs e)
     {
         _accountPopover.Popdown();
-        var saveFileDialog = Gtk.FileChooserNative.New(_controller.Localizer["NewAccount"], this, Gtk.FileChooserAction.Save, _controller.Localizer["Save"], _controller.Localizer["Cancel"]);
-        saveFileDialog.SetModal(true);
         var filter = Gtk.FileFilter.New();
         filter.SetName($"{_controller.Localizer["NickvisionMoneyAccount"]} (*.nmoney)");
         filter.AddPattern("*.nmoney");
-        saveFileDialog.AddFilter(filter);
-        saveFileDialog.OnResponse += async (sender, e) =>
+        if (Gtk.Functions.GetMinorVersion() >= 9)
         {
-            if (e.ResponseId == (int)Gtk.ResponseType.Accept)
+            var saveFileDialog = gtk_file_dialog_new();
+            gtk_file_dialog_set_title(saveFileDialog, _controller.Localizer["NewAccount"]);
+            var filters = Gio.ListStore.New(Gtk.FileFilter.GetGType());
+            filters.Append(filter);
+            gtk_file_dialog_set_filters(saveFileDialog, filters.Handle);
+            var fileHandle = IntPtr.Zero;
+            GAsyncReadyCallback callback = async (source, res, data) =>
```

Comment:
Shouldn't these be saved as instance variables to avoid that error "calling delegate on garbage collection object"...